adapters/klab.ogc/src/main/java/org/integratedmodelling/klab/stac/STACAssetParser.java

```diff
@@ -0,0 +1,17 @@
+package org.integratedmodelling.klab.stac;
+
+import java.util.Set;
+
+import kong.unirest.json.JSONObject;
+
+public class STACAssetParser {
+    final private static Set<String> SUPPORTED_MEDIA_TYPE = Set.of("image/tiff; application=geotiff; profile=cloud-optimized");
```

Comment:
What happens if the reported media type doesn't contain spaces between the parts? e.g. `text/plain;charset=UTF-8` [1] or in upper-case [2]

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types
[2] _Types, subtypes, and parameter names are case-insensitive. Parameter values are usually case-sensitive, but may be interpreted in a case-insensitive fashion depending on the intended use_ - https://en.wikipedia.org/wiki/Media_type

Reply:
Didn't know about that. Should be fixed before merging.